hub/core/experimental/cleanlab.py

```diff
@@ -0,0 +1,253 @@
+from hub.integrations import pytorch_module_to_skorch as to_skorch
+from hub.util.dataset import map_tensor_keys
+
+from cleanlab.filter import find_label_issues
+from cleanlab.rank import get_label_quality_scores
+
+from sklearn.model_selection import StratifiedKFold
+from sklearn.metrics import accuracy_score
+from sklearn.base import clone
+
+import numpy as np
+
+# TODO: Add default parameters and models
+# TODO: Add support for validation set
+# TODO: Add prune support to delete samples with label issues
+# TODO: Refactor as a class?
+# TODO: Add error checking for the len(tensors) after appending label issues tensors
+# TODO: Error checking, e.g. if number of samples not equal to number of labels in the initial dataset,
+# TODO: Add support for tensorflow models
+# TODO: Add types for the arguments
+# TODO: Create custom config for pip install (e.g. !pip install hub[’cleanlab’])
+# TODO: Add cleanlab kwargs
+# TODO: Add support to create valid and loss accuracy graphs
+# TODO: Add suppport for bounding boxes, task = 'classification' or 'segmentation'
+# TODO: Add tests for the functions
+# TODO: Use y as when calling fit() instead of passing the whole ds?
+# TODO: Use scikit learn shuffle for train set after getting the indices of train and holdout sets.
+# TODO: Instead of deleting label group, just overwrite it at the end if overwrite = True.
+# TODO: Add append parameter. If True, append label issues tensors to the dataset. If False, return label issues as np array.
+# TODO: Add guessed label to tensors
+# TODO: Add progress bar for the training
+# TODO: Add flag branch to move to a different branch.
+
+def get_dataset_tensors(dataset, tensors, dataloader_train_params, overwrite, verbose):
+    """
+    This function returns the tensors of a dataset. If a list of tensors is not provided,
+    it will try to find them in the dataloader_train_params in the transform. If none of
+    these are provided, it will iterate over the dataset tensors and return any tensors
+    that match htype 'image' for images and htype 'class_label' for labels. Additionally,
+    this function will also check if the dataset already has a label_issues group.
+    """
+
+    tensors_list = list(dataset.tensors)
+
+    if (
+        "label_issues/is_label_issue" in tensors_list
+        or "label_issues/label_quality_scores" in tensors_list
+    ):
+        if overwrite:
```

Comment:
Instead of doing this, I could also remove this code block and use `overwrite = True` when appending the tensors in `append_label_issues_tensors()`. What would be the most preferable way?